testsuite/tests/hidden_includes/c1_2.ocamlc.reference

```diff
@@ -0,0 +1,3 @@
+File "libc/c1.ml", line 1:
```

Comment:
Is it possible to name the reference files with something indicating which test case they correspond to? `c1_1`, `c1_2` are hard to associate with the actual case they're testing.